src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -0,0 +1,237 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import { struct } from "../../common/structs/struct";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { StackCardConfig } from "../../cards/types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+const cardConfigStruct = struct({
+  type: "string",
+  cards: ["any"],
+  title: "string?",
+});
+
+@customElement("hui-stack-card-editor")
+export class HuiAlarmPanelCardEditor extends LitElement
+  implements LovelaceCardEditor {
+  @property() public hass?: HomeAssistant;
+  @property() private _config?: StackCardConfig;
+  @property() private _selectedCard: number = 0;
+
+  public setConfig(config: StackCardConfig): void {
+    config = cardConfigStruct(config);
+    this._config = config;
```

Comment:
```suggestion
    this._config = cardConfigStruct(config);
```

Reply:
I saw it done this way in some other places, and assumed it had to do with error handling.
Will `this._config` have a sane value if `cardConfigStruct` throws an exception?

Reply:
it doesn't matter if it throws an error card is made.